bin/choco

```diff
@@ -1,14 +1,5 @@
 #!/bin/bash
 set -e
 
-# choco dumps a ton of stuff in the PWD/opt
-# This hack will clean it up automatically.
-function cleanup {
-	if [ $PWD != "/" ] && [ -d opt ]; then
-		rm -rf opt
-	fi
-}
-trap cleanup EXIT
-
 # Wrap the mono choco.exe command
-mono /opt/chocolatey/choco.exe "$@" --allow-unofficial
\ No newline at end of file
+mono /opt/chocolatey/console/choco.exe "$@" --allow-unofficial
```

Comment:
Why do we even need a choco script wrapping the real choco.exe ?

Reply:
For two reasons-
1. The exe cannot be executed directly, you need to run it through mono.
2. Chocolatey builds that are not signed with the official SNK need the `--allow-unofficial` argument passed to do anything more then show help. 

This wrapper makes so you can type just choco, instead of having to write out that whole line every time you want to run chocolatey. It also makes it a bit more consistent with the windows version.